components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, elRef, eventHandlers }) {
+  const { classList, display, width, backgroundColor, type, listItems, color, fontSize, padding } = component;
+
+  const items = listItems || [];
+
+  const onClickHandler = (e, itemContent, itemChildren) => {
+    e.stopPropagation();
+    eventHandlers.onItemClick({ itemContent, itemChildren });
+  };
+
+  const options = {
+    type: type === "ol" ? "ol" : "ul",
```

Comment:
why don't we just pass the `type` to the List? and let it decide what to do with the type

Reply:
The type initially passed from the user is applied to the external list, and then, if there are nested lists and the type is specified, then it is applied to the nested list, if not passed, then the external one is applied. This is the specifics of this component, in the course of development we decided to add such behavior.